<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <item>
     <widget class="QRadioButton" name="opcion1">
      <property name="text">
       <string>Opcion 1</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="opcion2">
      <property name="text">
       <string>Opcion 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="opcion3">
      <property name="text">
       <string>Opcion 3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="etiqueta">
      <property name="text">
       <string>Elija una opcion</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
